pkg/sqlstore/impl/system/sqlc.yaml

```diff
@@ -0,0 +1,29 @@
+version: "2"
+sql:
+  - schema: 
+      - "./migrations"
+      - "./schemas"
+    queries: "./queries"
+    engine: "sqlite"
+    gen:
+      go:
+        package: "db"
+        out: "./internal/db"
+        emit_prepared_queries: true
+        emit_interface: false
+        emit_exact_table_names: false
+        emit_empty_slices: false
+        emit_json_tags: false
+        json_tags_case_style: "camel"
+        sql_package: "database/sql"
+        overrides:
```

Comment:
I've added some type overrides to avoid a lot of casting inside [store.go](pkg/sqlstore/impl/system/store.go).